packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -250,6 +250,9 @@ function ColorPanelDropdown( {
 									icon={ resetIcon }
 									onClick={ () => {
 										resetValue();
+										if ( isOpen ) {
+											onToggle();
+										}
```

Comment:
It turns out this implementation is also necessary for the shadows panel:

https://github.com/user-attachments/assets/e2cf40bd-4c4d-432e-afa4-9493e525169c

Reply:
Perfect, will open a PR for that 🙌

Reply:
I ended up pushing the changes [here](https://github.com/WordPress/gutenberg/pull/67800/commits/63db22c3f81f47693145f357ea24604ae5bf3bd6).